src/java.desktop/share/classes/com/sun/java/swing/plaf/gtk/GTKLookAndFeel.java

```diff
@@ -1020,6 +1021,9 @@ public Object createValue(UIDefaults table) {
             "TabbedPane.selectedLabelShift", 3,
             "TabbedPane.font", new FontLazyValue(Region.TABBED_PANE),
             "TabbedPane.selectedTabPadInsets", new InsetsUIResource(2, 2, 0, 1),
+            "TabbedPane.selected", tabbedPaneBg,
```

Comment:
`TabbedPane.selected` already existed or newly added ? If newly added then it would be better to name it as `SelectedColor` or something similar.

Reply:
`TabbedPane.selected` is added as a new property. Property name is kept similar to Metal property (MetalTabbedPaneUI.java) to keep consistency.